README.md

```diff
@@ -7,4 +7,4 @@ toggled via toolbar icon.
 
 You can install it via [the Chrome Web Store](https://chrome.google.com/webstore/detail/wikimediadebug/binmakecefompkjggiklgjenddjoifbb).
 
-There is also a [Firefox version](https://addons.mozilla.org/firefox/addon/wikimedia-debug-header/).
+There is also a [Firefox version](https://github.com/wikimedia/FirefoxWikimediaDebug).
```

Comment:
The page at https://addons.mozilla.org/en-US/firefox/addon/wikimedia-debug-header/ links to the source repo on github, but I think linking people to the official releases is a nicer experience. Is there a strong motivation to do otehrwise?